lib/livebook/zta.ex

```diff
@@ -57,4 +57,16 @@ defmodule Livebook.ZTA do
   def put(name, value) do
     :ets.insert(__MODULE__, [{name, value}])
   end
+
+  def provider_name(provider_type) do
+    if provider_type != nil do
+      Livebook.Config.identity_providers()
+      |> Enum.find(fn provider ->
```

Comment:
We already have this function:

https://github.com/livebook-dev/livebook/blob/5e7f8a477a9e3154aac8c403cd237d9c73687bd1/lib/livebook_web/components/app_components.ex#L166-L168

Perhaps let's move it to `Livebook.Config`, and reuse?

Reply:
Good call. Done.

@jonatanklosko please review again to make sure it's ok. 🙂